ext/RMagick/extconf.rb

```diff
@@ -1,3 +1,5 @@
+lib_dir = File.expand_path('../../lib', File.dirname(__FILE__))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.